mac80211 fq has been found to cause a regression in multi-stream TCP
performance. Disable it until the cause has been found and fixed

--- a/include/net/fq_impl.h
+++ b/include/net/fq_impl.h
@@ -104,6 +104,9 @@ static struct fq_flow *fq_flow_classify(
 
 	lockdep_assert_held(&fq->lock);
 
+	/* HACK: disable fq for now until TCP issues are fixed */
+	return get_default_func(fq, tin, 0, skb);
+
 	hash = skb_get_hash_perturb(skb, fq->perturbation);
 	idx = reciprocal_scale(hash, fq->flows_cnt);
 	flow = &fq->flows[idx];
